symbolic-il2cpp/src/usym.rs

```diff
@@ -317,9 +320,8 @@ impl<'a> UsymSymbols<'a> {
     /// The ID of the assembly.
     ///
     /// This should match the ID of the debug symbols.
-    // TODO: Consider making this return debugid::DebugId
-    pub fn id(&self) -> &str {
-        self.id
+    pub fn id(&self) -> Option<DebugId> {
```

Comment:
You could do something like return `Result<DebugId, &str>` so you still return the string error if it could not be parsed.  If choosing that way probably should do the same for `arch`

Reply:
it is done 🎉